articles/openshift/oshift-sco.md

```diff
@@ -0,0 +1,201 @@
+---
+title: UKCloud for OpenShift Service Scope | UKCloud Ltd
+description: Outlines important details regarding UKCloud for OpenShift
+services: openshift
+author: Sue Highmoor
+reviewer: Steve Dixon
+lastreviewed: 08/07/2019
+toc_rootlink: Service Scope
+toc_sub1: 
+toc_sub2:
+toc_sub3:
+toc_sub4:
+toc_title: UKCloud for OpenShift Service Scope
+toc_fullpath: Service Scope/oshift-sco.md
+toc_mdlink: oshift-sco.md
+---
+
+# UKCloud for OpenShift Service Scope
+
+## About this document
+
+This document is for customers considering purchasing our UKCloud for OpenShift service.
+
+It describes the boundaries of the service, along with the division of responsibilities between UKCloud and the customer to facilitate the changes required.
+
+## About UKCloud for OpenShift
+
+UKCloud for OpenShift is UKCloud’s OpenShift powered Kubernetes Platform as a Service (PaaS).
+
+This service is designed for customers who want to:
+
+- Deliver new software features faster, focusing on customer value rather than infrastructure complexities
+
+- Cope with unpredictable spikes in traffic and customer demand
+
+- Realise more value from your DevOps team by allowing them to focus on the 'Dev' rather than 'Ops'
+
+- Create a standard approach to deployment, ensuring consistency across all users
+
+- Perform platform maintenance with less disruption to applications
+
+- Try out new ideas in a low-risk sandbox environment
+
+- Remove the cost and complexity of provisioning, managing and scaling the underlying infrastructure, OS and middleware components
+
+- Avoid lock-in — you can deploy OpenShift on a wide choice of IaaS platforms
+
+## OpenShift technical features available in UKCloud for OpenShift
+
+The following OpenShift technical features are available with UKCloud for OpenShift:
+
+- UKCloud for OpenShift is built upon Red Hat OpenShift Container Platform v3.11
+
+- Based upon OKD 3.11 with an Enterprise security and stability wrap
+
+- Kubernetes 1.11
+
+- UKCloud for OpenStack is the underlying Infrastructure-as-a-Service (IaaS) platform
+
+## Default setup
+
+UKCloud for OpenShift is delivered as a private Platform-as-a-Service (PaaS).
+
+Each customer will be provided with an isolated OpenShift cluster within which they can build their containerized solutions.
```

Comment:
containerized or containerised? article has both spellings

Reply:
Should be containerised... bloody spellcheck!